hugegraph-llm/src/hugegraph_llm/demo/rag_web_demo.py

```diff
@@ -517,27 +524,56 @@ def reranker_settings(reranker_type):
 }"""
 
         with gr.Row():
-            input_file = gr.File(
-                value=[os.path.join(resource_path, "demo", "test.txt")],
-                label="Docs (multi-files can be selected together)",
-                file_count="multiple",
-            )
-            input_schema = gr.Textbox(value=schema, label="Schema")
-            info_extract_template = gr.Textbox(value=SCHEMA_EXAMPLE_PROMPT, label="Info extract head")
             with gr.Column():
-                mode = gr.Radio(
-                    choices=["Test Mode", "Import Mode", "Clear and Import", "Rebuild Vector"],
-                    value="Test Mode",
-                    label="Build mode",
-                )
-                btn = gr.Button("Build Vector/Graph RAG")
+                with gr.Tab("file") as tab_upload_file:
+                    input_file = gr.File(
+                        value=[os.path.join(resource_path, "demo", "test.txt")],
+                        label="Docs (multi-files can be selected together)",
+                        file_count="multiple",
+                    )
+                with gr.Tab("text") as tab_upload_text:
+                    input_text = gr.Textbox(value="", label="Doc(s)", lines=20, show_copy_button=True)
+            input_schema = gr.Textbox(value=schema, label="Schema", lines=15, show_copy_button=True)
+            info_extract_template = gr.Textbox(value=SCHEMA_EXAMPLE_PROMPT, label="Info extract head", lines=15,
+                                               show_copy_button=True)
+            out = gr.Textbox(label="Output", lines=15, show_copy_button=True)
+
         with gr.Row():
-            out = gr.Textbox(label="Output", show_copy_button=True)
-        btn.click(  # pylint: disable=no-member
-            fn=build_kg,
-            inputs=[input_file, input_schema, info_extract_template, mode],
-            outputs=out,
+            vector_index_btn0 = gr.Button("Get Vector Index Info")
```

Comment:
As the function button changes, maybe we should consider the corresponding function description should also be updated?

![image](https://github.com/user-attachments/assets/a5a49297-1974-4256-91a2-329a9cbb37b2)

Reply:
The modification has been completed